src/components/Comment/component.js

```diff
@@ -75,34 +77,32 @@ export default class Comment extends React.Component {
 
     if (editing) return <CommentForm commentId={comment.id} close={closeEdit} />
 
-    const { image } = comment
-
     return <div className='comment' data-comment-id={comment.id}>
-      {canEditComment(currentUser, comment, community) &&
+      {notPending && canEditComment(currentUser, comment, community) &&
         <Dropdown alignRight toggleChildren={<Icon name='More' />}>
           {!image && <li><a onClick={edit}>Edit</a></li>}
           <li><a onClick={remove}>Remove</a></li>
         </Dropdown>}
-      <a name={`comment-${comment.id}`} />
+      {notPending && <a name={`comment-${comment.id}`} />}
       <Avatar person={person} showPopover />
       <div className='content'>
         {image && <div className='text'>
           <A to={`/u/${person.id}`} className='name'>{person.name}</A>
         </div>}
         {image && <a onClick={() => showImage(image.url, location.pathname, isMobile)}>
-          <img className='thumbnail' src={image.thumbnail_url} />
+          <ImageWithFallback useClass='thumbnail' preferredSrc={image.thumbnail_url} fallbackSrc={image.url} />
```

Comment:
If we use this for all comments, doesn't that cause the large image size to be loaded for all existing comments with images? we'd definitely want to avoid that...